src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -973,12 +1171,45 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
         )
     }
 
+    getEmptyStateSubtitle = () => {
+        if (this.props.triggerType === TriggerTypeMap.automatic) {
+            return `Deployment to ${this.props.envName} is set to Automatic. Deployment of an image is initiated as soon as it receives the required number of approvals.`
+        } else if (this.state.isRollbackTrigger) {
+            return 'Approved images which have been previously deployed will be available here for rollback.'
+        } else {
+            return 'Approved images will be available here for deployment.'
+        }
+    }
+
+    renderEmptyState = () => {
+        if (this.props.userApprovalConfig?.requiredCount > 0) {
+            return (
+                <>
+                    <div className="material-list__title pt-16 pl-20 pr-20">Approved images</div>
+                    <EmptyView title="No image available" subTitle={this.getEmptyStateSubtitle()} imgSrc={noartifact} />
```

Comment:
Please use generic empty state

Reply:
As discussed, this will be taken care during UX/Bug fixes.